solution-base/mongodb/charts/mongodb/custom-values.yaml

```
@@ -493,7 +493,7 @@ metrics:
 
     ## Used to pass Labels that are used by the Prometheus installed in your cluster to select Service Monitors to work with
     ## ref: https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#prometheusspec
-    additionalLabels: {}
+    additionalLabels: { app.kubernetes.io/name: '' }
```

Could use the more yaml syntax, might need to quote the key not quite sure
```suggestion
    additionalLabels: 
        app.kubernetes.io/name: ''
```

---

👍